docs/Non-Critical_Notes/Configuration_Variables.md

```diff
@@ -0,0 +1,69 @@
+# Configuration Variables List
+
+## Variables
+
+### Integer Configuration
+
+**1) `TASK_heartbeat_period_ms`**: The frequency, in ms, at which the `TASK_DEBUG_print_heartbeat` telecommand is logging messages.
+
+**2) `CONFIG_freertos_min_remaining_stack_percent`**:  
+ The percentage of the stack space that should remain free. If the free space falls below this percentage, a warning will be logged.
+
+**3) `CONFIG_int_config_variables[]`**:  
+ An array of objects, where each object contains:
+
+- The variable name (const char \*)
+- int value (uint32_t \*)
+
+**4) `CONFIG_int_config_variables_count`**:  
+ The number of entries in the `CONFIG_int_config_variables[]` array.
+
+---
+
+### String Configuration
+
+**5) `CONFIG_str_config_variables[]`**:  
```

Comment:
This isn't a configuration variable; it's an index of all the actual configuration variables. I don't think there's really a string configuration variable registered yet.